app/client/keybase/debug/keystore.go

```diff
@@ -0,0 +1,131 @@
+package debug
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/pokt-network/pocket/app/client/keybase"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	validatorNumber     = 999
```

Comment:
Rename to `numValidators` or `validatorCount` and add a comment `// The number of pre-generated keys in privateKeysYamlFile`